build/config/ports-installer.pyd

```diff
@@ -32,7 +32,8 @@ ports += "lang/python3"
 ports += "lang/python"
 ports += "databases/sqlite3"
 ports += "databases/py-sqlite3"
-ports += "databases/py-bsddb3"
+# TODO: Is berkeleydb even used?
+ports += "databases/py-berkeleydb"
```

Comment:
I think this was for John's legacy user / group caching in FN prior to 11.3.

Reply:
But this is for the installer specifically, which is odd.